build.gradle

```diff
@@ -133,6 +133,8 @@ dependencies {
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-batch'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
 
+    implementation group: 'org.apache.pdfbox', name: 'pdfbox', version: '3.0.2'
+    implementation group: 'org.apache.pdfbox', name: 'pdfbox-io', version: '3.0.2'
```

Comment:
why do we need these?

Reply:
These were used by the pdf parser, with the change to AutoDetectParser they can likely be removed.